app/main/views/frameworks.py

```diff
@@ -330,3 +332,110 @@ def get_framework_interest(framework_slug):
     supplier_ids = [supplier_framework.supplier_id for supplier_framework in supplier_frameworks]
 
     return jsonify(interestedSuppliers=supplier_ids), 200
+
+
+@main.route('/frameworks/transition-dos/<string:framework_slug>', methods=['POST'])
+def transition_dos_framework(framework_slug):
```

Comment:
Can we add a brief docstring to explain the view (the description from this PR would be ideal)?

Reply:
Yep, will do.